modules/textures/test/test-utils/check-compressed-texture.js

```diff
@@ -0,0 +1,12 @@
+export function checkCompressedTexture(imageData, testCase, t) {
```

Comment:
I usually put `t` parameter first in these functions.

Reply:
Done